### DISCUSSION TITLE
Reposition a combo box popup

### DISCUSSION THREAD

**Discussion by no-materials** [emilk/egui#976] [Q&A]
Is there any way to move a Combobox popup to be positioned above the widget? I do not want to render it outside the parent's cliprect which is fixed by https://github.com/luxtorpeda-dev/egui/commit/96c3766cd02814a970b00d8e866be7e1946e8324.

**Comment by emilk** [comment]
There is an open issue to fix this: https://github.com/emilk/egui/issues/693
